src/main/org/codehaus/groovy/tools/javac/JavaStubGenerator.java

```diff
@@ -925,7 +931,8 @@ private void printImports(PrintWriter out, ClassNode classNode) {
         imports.addAll(Arrays.asList(ResolveVisitor.DEFAULT_IMPORTS));
 
         for (Map.Entry<String, ImportNode> entry : moduleNode.getStaticImports().entrySet()) {
-            imports.add("static "+entry.getValue().getType().getName()+"."+entry.getKey());
+            if (entry.getKey().equals(entry.getValue().getFieldName()))
```

Comment:
I'm rethinking my comment now.  I thought it'd be better to include the non-aliased name, but that could cause a problem if it was aliased to work around a name conflict.  I'm thinking now what you did is the right thing.